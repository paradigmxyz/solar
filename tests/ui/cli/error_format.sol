//@ revisions: ascii unicode short
//@[ascii] compile-flags: --error-format-human=ascii
//@[unicode] compile-flags: --error-format-human=unicode
//@[short] compile-flags: --error-format-human=short

// SPDX-License-Identifier: UNLICENSED
pragma solidity ^0.8.13;

contract TestError {
    uint256 public number;

    function test() public {
        uint256 x = 1;
        uint256 y = 2;
        uint256 z = x + y
    } //~ ERROR: expected one of
}
