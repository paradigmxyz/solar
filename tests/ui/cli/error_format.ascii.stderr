error: expected one of `(`, `.`, `;`, `?`, `[`, or `{`, found `}`
  --> ROOT/tests/ui/cli/error_format.sol:LL:CC
   |
LL |         uint256 z = x + y
   |                          - expected one of `(`, `.`, `;`, `?`, `[`, or `{`
LL |     }
   |     ^ unexpected token

error: aborting due to 1 previous error

